components/calendar/src/helpers.rs

```diff
@@ -26,6 +37,8 @@ pub const fn quotient(n: i32, d: i32) -> i32 {
         a - 1
     }
 }
+
+/// [`quotient`] for 64-bit inputs
```

Comment:
Isn't this `n.div_euclid(d)`?

Reply:
Good observation, but this is pre-existing code. Added comments and a test.